uk.co.compucorp.civicrm.tasksassignments/views/contact/documents.html

```diff
@@ -73,7 +73,10 @@
 
 <script type="text/ng-template" id="document.html">
   <td>
-    <a ng-href="{{fileUrl}}" ng-click="(!document.file_count || document.file_count == '0') && modalDocument('edit', document, $event);" ng-bind="cache.documentType.obj[document.activity_type_id]"></a>
+    <a ng-href="{{fileUrl}}"
+      ng-click="(!document.file_count || document.file_count == '0') && modalDocument('edit', document, $event);">
+      {{cache.documentType.obj[document.activity_type_id]}}
```

Comment:
This is different from ng-bind, are you sure this will not produce unexpected behaviour?

Reply:
How is it different? As far as I know this is the same. Output also looks the same as in the previous version:
![civihr_staff compucorp co uk hr17 9000](https://user-images.githubusercontent.com/1642119/35393082-4d6dd31c-01ba-11e8-8392-01068a80a637.png)

The reason for moving it it's because it's more explicit to place the element content inside of it using interpolation instead of using `ng-bind`.

Reply:
Right. However, they are not the same. Please see https://stackoverflow.com/questions/16125872/angularjs-why-ng-bind-is-better-than-in-angular and find an answer where it starts with the word "Visibility:"